enhancements/windows-containers/health-management.md

```diff
@@ -221,8 +227,17 @@ For Node configuration WICD will have two separate responsibilities:
 
 #### Bootstrap command
 
-When run with the `bootstrap` command, WICD will do the steps required to ensure that Node is created
-for the instance. This functionality is currently implemented within WMCB with the `init-kubelet` command.
+When run with the `bootstrap` command, WICD will do the steps required to ensure that Node is created for the instance. 
+When containerd is the container runtime, the `bootstrap` command will have the added responsibility of starting 
```

Comment:
By the time we implement WHM, the containerd always going to be the runtime.  Do we still need to call this out? This brings other questions like how do we handle other runtimes? Should we mention containerd will be the only supported runtime by WHM?

Reply:
Since we will continue using docker through 5.x and only switching to containerd officially in 6.0.0, I think it is better to call out responsibilities in both cases. I don't want to restrict the scope of the enhancement based on an arbitrary timeline. Assume we finish health management work quickly and release it as part of 5.1 -- then the enhancement would be out of sync and we would have to come back again to update it.